etc/apparmor/firejail-default

```diff
@@ -49,6 +49,10 @@ owner /{,run/firejail/mnt/oroot/}{,var/}run/firejail/mnt/trace w,
 owner /{,run/firejail/mnt/oroot/}{,var/}run/user/[0-9]*/** w,
 owner /{,run/firejail/mnt/oroot/}{run,dev}/shm/** w,
 
+# Allow writing to /var/mail and /var/spool/mail (for mail clients)
+# Uncomment to enable
+#owner /{,var/}{mail,spool/mail}/** w,
```

Comment:
this also allows /mail and /spool/mail, right? (as the var is optional)
consider changing it to `/var/{mail,spool/mail}/**`

Reply:
Nice catch, thanks!